### PR TITLE
Can we add zIndex to dialog to allow us to control the order they are rendered

### PR DESCRIPTION
**Pull request by Hoskins355**

### CODE REVIEW

File: src/Dialog.js [hunk 1]
```diff
@@ -110,7 +111,7 @@ export default class Dialog extends Component {
     const {center} = this.props
     const width = this.props.default.width
     const x = center && width ? (window.innerWidth / 2) - (width / 2) : this.props.default.x
-    const defaultProps = {...this.props.default, x}
+    const defaultProps = {...this.props.default, x, this.props.zIndex}
```
Comment (dadamssg) [inline src/Dialog.js]: i think you meant this:

```js
const defaultProps = {...this.props.default, x, zIndex: this.props.zIndex}
```

Comment (dadamssg) [reply src/Dialog.js]: actually i'm pretty sure this code doesn't matter since your `zIndex` prop is only used directly on line 83.

Comment (Hoskins355) [reply src/Dialog.js]: I agree I dont think it is needed

**Pull request merged**